src/main/java/org/spongepowered/api/CatalogTypes.java

```diff
@@ -0,0 +1,129 @@
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package org.spongepowered.api;
+
+import org.spongepowered.api.attribute.Attribute;
+import org.spongepowered.api.attribute.Operation;
+import org.spongepowered.api.block.BlockType;
+import org.spongepowered.api.block.tile.TileEntityType;
+import org.spongepowered.api.block.tile.data.BannerPatternShape;
+import org.spongepowered.api.effect.particle.ParticleType;
+import org.spongepowered.api.effect.sound.SoundType;
+import org.spongepowered.api.entity.EntityInteractionType;
+import org.spongepowered.api.entity.EntityType;
+import org.spongepowered.api.entity.hanging.art.Art;
+import org.spongepowered.api.entity.living.animal.HorseColor;
+import org.spongepowered.api.entity.living.animal.HorseStyle;
+import org.spongepowered.api.entity.living.animal.HorseVariant;
+import org.spongepowered.api.entity.living.animal.OcelotType;
+import org.spongepowered.api.entity.living.animal.RabbitType;
+import org.spongepowered.api.entity.living.monster.SkeletonType;
+import org.spongepowered.api.entity.living.villager.Career;
+import org.spongepowered.api.entity.living.villager.Profession;
+import org.spongepowered.api.entity.player.gamemode.GameMode;
+import org.spongepowered.api.item.CoalType;
+import org.spongepowered.api.item.CookedFish;
+import org.spongepowered.api.item.DyeColor;
+import org.spongepowered.api.item.Enchantment;
+import org.spongepowered.api.item.FireworkShape;
+import org.spongepowered.api.item.Fish;
+import org.spongepowered.api.item.GoldenApple;
+import org.spongepowered.api.item.ItemType;
+import org.spongepowered.api.item.inventory.equipment.EquipmentType;
+import org.spongepowered.api.potion.PotionEffectType;
+import org.spongepowered.api.scoreboard.Visibility;
+import org.spongepowered.api.scoreboard.critieria.Criterion;
+import org.spongepowered.api.scoreboard.objective.displaymode.ObjectiveDisplayMode;
+import org.spongepowered.api.stats.Statistic;
+import org.spongepowered.api.stats.StatisticFormat;
+import org.spongepowered.api.stats.StatisticGroup;
+import org.spongepowered.api.stats.achievement.Achievement;
+import org.spongepowered.api.text.chat.ChatType;
+import org.spongepowered.api.text.format.TextColor;
+import org.spongepowered.api.text.selector.SelectorType;
+import org.spongepowered.api.util.rotation.Rotation;
+import org.spongepowered.api.world.DimensionType;
+import org.spongepowered.api.world.GeneratorType;
+import org.spongepowered.api.world.biome.BiomeType;
+import org.spongepowered.api.world.difficulty.Difficulty;
+import org.spongepowered.api.world.weather.Weather;
+
+/**
+ * Enumeration of all known {@link CatalogType}s for autocompletion when using the
+ * {@link GameRegistry#getType(Class, String)} and {@link
+ * GameRegistry#getAllOf(Class)}.
+ */
+public final class CatalogTypes {
```

Comment:
You missed one :stuck_out_tongue:. CatalogTypes is a catalog of catalog types, too!